apps/mobile/src/app/(home)/settings/wallet/configure/[wallet]/index.tsx

```diff
@@ -1,9 +1,9 @@
-import { useEffect, useRef } from 'react';
+import { useEffect, useRef, useState } from 'react';
```

Comment:
This file is pretty big and we are adding more to it. Maybe we can split it up into `.layout` components?